<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeSettingsPane</class>
 <widget class="QWidget" name="ThemeSettingsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-195</y>
        <width>715</width>
        <height>783</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="coloursWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="AccentColourPicker" name="accentBlue" native="true">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="AccentColourPicker" name="accentGreen" native="true">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="AccentColourPicker" name="accentOrange" native="true">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="AccentColourPicker" name="accentPink" native="true">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Accent Colour</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="baseColourComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Light</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Change the colours of the interface</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Base Colour</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>COLOURS</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="tConditionalWidget" name="windowBordersConditonalWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="windowBordersWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>WINDOW BORDERS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>If the window borders don't look correct, we can fix that up for you. This will also change your window borders in KDE, and any other desktop that uses KWin for the window manager.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="setWindowBordersButton">
                 <property name="text">
                  <string>Set Window Borders</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="fontsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Fixed Width Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Interface Font</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFontComboBox" name="interfaceFont">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>FONTS</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QFontComboBox" name="fixedFont">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="fontFilters">
             <set>QFontComboBox::MonospacedFonts</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="interfaceFontSize">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="fixedFontSize">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="widgetsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Set the appearance of buttons, text boxes and other interface elements</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Widget Theme</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>WIDGETS</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="widgetThemeBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="4" column="1">
              <widget class="QProgressBar" name="progressBar_2">
               <property name="maximum">
                <number>0</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Button</string>
                 </property>
                 <property name="icon">
                  <iconset theme="dialog-ok">
                   <normaloff>.</normaloff>.</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>Radio Button 2</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Using the theme set above, interface elements will look like this:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>Radio Button 1</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>Check Box 2</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>DEMO</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Check Box 1</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QLineEdit" name="lineEdit">
               <property name="placeholderText">
                <string>Type here!</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QWidget" name="effectsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Bar Opacity</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="opacitySlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Transparency Effects</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>EFFECTS</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="opacitySliderLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>100%</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="tSwitch" name="transparencySwitch">
            <property name="text">
             <string notr="true">TransparencySwitch</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">tswitch.h</header>
  </customwidget>
  <customwidget>
   <class>AccentColourPicker</class>
   <extends>QWidget</extends>
   <header>settings/accentcolourpicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tConditionalWidget</class>
   <extends>QWidget</extends>
   <header location="global">tconditionalwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>baseColourComboBox</tabstop>
  <tabstop>accentBlue</tabstop>
  <tabstop>accentGreen</tabstop>
  <tabstop>accentOrange</tabstop>
  <tabstop>accentPink</tabstop>
  <tabstop>interfaceFont</tabstop>
  <tabstop>interfaceFontSize</tabstop>
  <tabstop>fixedFont</tabstop>
  <tabstop>fixedFontSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
